docs/docs/guides/trainings/supportservice/application.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Application Support Trainings
+---
+
+## ZITADEL Developer
```

Comment:
I find "developer" confusing... a training for developers might be geared towards the API and not the ZITADEL internals.

Maybe we should call this ZITADEL Internals?

Reply:
```suggestion
## ZITADEL Support
```